web/language/migrations/0095_placename_audio.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.4 on 2019-11-13 20:17
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('language', '0094_community_nation_id'),
+    ]
+
+    operations = [
```

Comment:
@DenisCalixto you'll need a python code migration to move the old audio_file into the new object, when it exists.

https://medium.com/quick-code/how-to-create-custom-migrations-in-django-610718f49ae2

Reply:
I will work on that but Daniel does not need to wait for that to keep on working on the entities which does not have anything to migrate, so I am merging it now to unblock him.